consensus/src/network.rs

```diff
@@ -212,6 +212,7 @@ impl NetworkSender {
         self.send(msg, recipients).await
     }
 
+    #[cfg(feature = "failpoints")]
```

Comment:
why do you need this line?

Reply:
to suppress compiler warning about unused code